### ISSUE TITLE
USER ISSUE: Brick Cube has bad UV Map

### ISSUE THREAD

**Issue by occultsoftworks** [StrangeLoopGames/EcoIssues#20844]
\
**User Id: (Not logged user)**
**Version:** 0.9.2.4 beta release-156

**Steps to Reproduce:**
The Brick Cubes don't tile with each other, and are offset at the edges. It looks like the UV needs to be adjusted. This is most noticable at the corner vertical edges.	

**Expected behavior:**
It should tile with itself and neighbouring tiles.	

**Actual behavior:**
Does not tile with itself and neighbouring tiles.	

**Do you have mods installed? Does the issue happen when no mods are installed?:**
I don't.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by occultsoftworks** [comment]
![image](https://user-images.githubusercontent.com/28337733/110192819-8a46e800-7dfe-11eb-8012-18e2fde1c515.png)

**Comment by Daniel-Magulado** [comment]
Hi, thank you for making a report about this.

Furthermore, the same issue can also be observed with corner wall-type bricks, aqueduct, and plain brick.

Brick Cube
![brick.jpg](https://images.zenhubusercontent.com/5ffb75c5aa5c9e2a1278e024/10934a34-98ac-46df-969e-86b1670418ec)

Corner Wall
![brick 2.jpg](https://images.zenhubusercontent.com/5ffb75c5aa5c9e2a1278e024/affc2948-dbd7-477c-a4f0-4845b8b7e2fe)

Aqueduct
![brick3.jpg](https://images.zenhubusercontent.com/5ffb75c5aa5c9e2a1278e024/3e38d589-ff14-4735-a568-5cc403753141)

Plain Brick
![brick4.jpg](https://images.zenhubusercontent.com/5ffb75c5aa5c9e2a1278e024/9a9e8947-b98f-4927-ab29-255193350190)

**Comment by ThePiachu** [comment]
Yeah, even if you have a uniform brick wall you can see the repeating textures so much

![image](https://user-images.githubusercontent.com/964559/117536409-b07c3600-afaf-11eb-8c51-34b823b5d53e.png)

**Comment by ThePiachu** [comment]
Speaking of Brick textures - there is a very noticeable colour difference between some Brick forms:

![image](https://user-images.githubusercontent.com/964559/118343184-421e0300-b4dc-11eb-8dd7-c58bf71db3f9.png)

One is rather more yellow / orange than the rest.
